views/index.html.twig

```diff
@@ -1,34 +1,81 @@
 {% extends "layout.html.twig" %}
 
+
 {% block header %}
 {% endblock %}
 
+
 {% block content %}
-    <div class="logo">
-        <img src="{{ app.request.basePath }}/img/{{ logo }}">
-        <h1 class="slogan">
-            Dependency Manager for PHP
-        </h1>
-        <p class="latest">Latest: v{{ latestStable }}</p>
-    </div>
-
-    <p class="buttons">
-        <a href="{{ path('docs.view', {page: '00-intro.md'}) }}">Getting Started</a>
-        <a href="{{ path('download') }}">Download</a>
-    </p>
-    <p class="buttons">
-        <a href="{{ path('docs') }}">Documentation</a>
-        <a href="https://packagist.org/">Browse Packages</a>
-    </p>
-    <p class="buttons">
-        <a href="https://github.com/composer/composer/issues">Issues</a>
-        <a href="https://github.com/composer/composer">GitHub</a>
-    </p>
+
+    <section role="banner">
+        <div class="logo">
+            <figure>
```

Comment:
Well, caption is optional, but this still adds semantic structural info, right?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure

Reply:
I'll close that unless you really feel that it's a real drawback instead of a progress, @stof. I'm pretty sure this goes forward. Feel free to reopen.